code/modules/research/machinery/_production.dm

```diff
@@ -172,8 +172,9 @@
 		if(!reagents.has_reagent(R, D.reagents_list[R]*print_quantity/coeff))
 			say("Not enough reagents to complete prototype[print_quantity > 1? "s" : ""].")
 			return FALSE
-	var/total_cost = LATHE_TAX
+	var/total_cost = 0
 	if(is_station_level(z) && isliving(usr)) //We don't block purchases off station Z.
+		total_cost = LATHE_TAX
```

Comment:
In an ironic twist of fate, won't this run into the opposite problem as #66566, where golems on lower level icebox will be charged for using their lathes?

Reply:
how would you recommend setting this up such that golems won't be charged for their lathes but people on the top level of icebox will?

I forget, can golems acquire credits by redeeming mining points for money?

Reply:
Yes mining points can be used for money, but if I remember correctly, golems have no actual bank accounts associated with their IDs, so they can't actually put the money in to pay the tax